mitodl/mit-open
# Updated OLL csv file

### What are the relevant tickets?
Related to https://github.com/mitodl/hq/issues/5350

### Description (What does it do?)
Updates the OLL csv file with better descriptions

### Screenshots (if appropriate):

![Screenshot 2024-09-13 124330](https://github.com/user-attachments/assets/3fcb3fc2-29ab-4bb6-bcd3-09d630111b1d)

### How can this be tested?
- Run `./manage.py backpopulate_oll_data`, it should complete successfully.
- Go to http://open.odl.local:8062/search?+Ethics%22=&platform=oll&q=%22Introduction+to+Ethics%22, click the course to open the drawer, check that the description includes some bullet points.
